### PR TITLE
deleteOption

### PR DESCRIPTION
**Pull request by imsandip2085**

### CODE REVIEW

File: polling-app/src/Components/UpdatePoll/DeleteOptionConfirmationBox.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+
+class DeleteOptionConfirmationBox extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            deleteOptionShow: false,
+        }
+    }
+    handleDeleteYes = () => {
```
Comment (nodeexcel) [inline polling-app/src/Components/UpdatePoll/DeleteOptionConfirmationBox.js]: what is use of this

File: polling-app/src/Components/UpdatePoll/DeletePollConformationBox.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import UpdatePoll from './UpdatePoll';
+
+class DeletePollConfirmationBox extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            deletePollShow: false,
+        }
+    }
+    handleDeleteYes = () => {
```
Comment (nodeexcel) [inline polling-app/src/Components/UpdatePoll/DeletePollConformationBox.js]: what is this for

Comment (imsandip2085) [reply polling-app/src/Components/UpdatePoll/DeletePollConformationBox.js]: it is nothing i forget delete it

File: polling-app/src/Components/UpdatePoll/DeleteOptionConfirmationBox.js (hunk 1 above)
Comment (imsandip2085) [reply polling-app/src/Components/UpdatePoll/DeleteOptionConfirmationBox.js]: it is nothing i forget delete it

**Pull request merged**